static_report/src/App.tsx

```diff
@@ -7,6 +7,29 @@ import { useHashLocation } from './hooks/useHashLcocation';
 import { SingleReportList } from './components/SingleReport/SRList';
 import { ComparisonReportList } from './components/ComparisonReport/CRList';
 
+import * as Sentry from '@sentry/browser';
+import { BrowserTracing } from '@sentry/tracing';
+
+const sentry_dns = window.PIPERIDER_METADATA.sentry_dns || null;
```

Comment:
js convention: please use `camelCase` over `snake_case`, unless it's `const` or environment variables